Functional programming/Problem 1/partial_application.js

```diff
@@ -0,0 +1,15 @@
+function F() {
```

Comment:
It's clear, but under the terms of the problem 1 I need to implement function *F*.
> **Problem 1: Partial Application**
> Implement function *F* that allows to do…